aws-analytics-pinpoint/build.gradle

```diff
@@ -16,6 +16,7 @@
 apply plugin: 'com.android.library'
 apply from: rootProject.file("configuration/checkstyle.gradle")
 apply from: rootProject.file("configuration/publishing.gradle")
+apply plugin: 'kotlin-android'
```

Comment:
Although I am a big fan of Kotlin, I do agree with others that we probably need more planning before including Kotlin on amplify-android.  There are many developer customers out there with various app environments with varying degrees of Kotlin integration in their apps, some of which may not tolerate sudden inclusion of Kotlin plugin in their app.  So I'm thinking that we probably need to coordinate with PM to do at least some pre-announcement of upcoming Kotlin inclusion and increase major version number.

Reply:
Are you pointing out to Kotlin plugin in this gradle or any kotlin class?